### ISSUE TITLE
Resolve a CSS issue with Chrome

### ISSUE THREAD

**Issue by Daniel-Miller** [InSite/Lotus#12]
Chrome seems to be the only browser with this issue. Notice the radio buttons render as partially filled blue circles. Firefox, IE, Safari, etc. render them correctly as open circles.

<img width="570" alt="screen shot 2016-05-10 at 3 34 06 pm" src="https://cloud.githubusercontent.com/assets/5367871/15163347/2b1fab82-16c5-11e6-8c21-300e6f326257.png">

**Comment by Daniel-Miller** [comment]
We should also modify our CSS so that when you answer a question incorrectly, the icon displayed inside the radio button is fa-times.  We should use fa-check only when the user has answered the question correctly.  I have implemented this in the Iris quiz module, if you need sample code for reference.

**Comment by Oleg-Terzi** [comment]
Done.


> Notice the radio buttons render as partially filled blue circles.

I discovered that in this way the Awesome Bootstrap Checkbox shows that an input is in indeterminate state:
https://developer.mozilla.org/en-US/docs/Web/CSS/:indeterminate
It seems that Chrome browser considers that if there are no selection for group of radio buttons then the radio buttons are in indeterminate state. At the same time all other browsers apply indeterminate state only to checkboxes and only if developer manually sets to Indeterminate property of input the True value.
I changed a little the styling for indeterminate state:
![img_1](https://cloud.githubusercontent.com/assets/5382331/15182455/345b4f00-1795-11e6-8881-651c2e9e2f07.png)

And I removed indeterminate styling for radio buttons so now it renders identically for all browsers.
